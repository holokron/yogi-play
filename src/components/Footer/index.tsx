import * as React from "react";

export interface Props {
  children?: any;
}

export default function Footer({ children = null }: Props) {
  return (
    <p className="text-muted text-center mt-4">
      {children}
      &copy;&nbsp;
      <a className="text-muted" href="https://github.com/holokron">
        Holokron
      </a>
    </p>
  );
}
