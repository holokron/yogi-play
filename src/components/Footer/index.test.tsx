import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import Footer from ".";

describe("@components/Footer", () => {
  it("renders correctly", () => {
    const tree = render(<Footer />);

    expect(tree).toMatchSnapshot();
  });
});
