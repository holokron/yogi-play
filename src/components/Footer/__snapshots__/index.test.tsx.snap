// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@components/Footer renders correctly 1`] = `
<p
  className="text-muted text-center mt-4"
>
  © 
  <a
    className="text-muted"
    href="mailto:michalv8@gmail.com"
  >
    Yogi
  </a>
</p>
`;
